vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -392,6 +395,23 @@
     // </editor-fold>
 
     // -----------------------------------------------------------------------------------------------------
+    // IAP
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="In-App Purchases">
+    @GET("products")
```

Comment:
Are the available products connected to the User in any way or are they generally available to anyone?

Reply:
They are specific to platform not user.

Reply:
Cool. I guess my bigger-picture question still applies here though... Is this an endpoint that can be configurable from somewhere in the API or is it a true top-level endpoint?